airbyte-integrations/connectors/source-metabase/source_metabase/schemas/databases.json

```
@@ -0,0 +1,118 @@
+{
+  "type": ["null", "object"],
+  "properties": {
+    "description": {
+      "type": ["null", "string"]
+    },
+    "features": {
+      "type": ["null", "array"],
+      "items": {
+        "type": ["null", "string"]
+      }
+    },
+    "cache_field_values_schedule": {
+      "type": ["null", "string"]
+    },
+    "timezone": {
+      "type": ["null", "string"]
+    },
+    "auto_run_queries": {
+      "type": ["null", "boolean"]
+    },
+    "metadata_sync_schedule": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "settings": {
+      "type": ["null", "string"]
+    },
+    "caveats": {
+      "type": ["null", "string"]
+    },
+    "creator_id": {
+      "type": ["null", "integer"]
+    },
+    "is_full_sync": {
+      "type": ["null", "boolean"]
+    },
+    "updated_at": {
+      "type": ["null", "string"]
+    },
+    "native_permissions": {
+      "type": ["null", "string"]
+    },
+    "cache_ttl": {
+      "type": ["null", "integer"]
+    },
+    "details": {
+      "type": ["null", "object"],
+      "properties": {
+        "project-id": {
+          "type": ["null", "string"]
+        },
+        "service-account-json": {
+          "type": ["null", "string"]
+        },
+        "dataset-filters-type": {
+          "type": ["null", "string"]
+        },
+        "dataset-filters-patterns": {
+          "type": ["null", "string"]
+        },
+        "cloud-ip-address-info": {
+          "type": ["null", "string"]
+        },
+        "advanced-options": {
+          "type": ["null", "boolean"]
+        },
+        "project-id-from-credentials": {
+          "type": ["null", "string"]
+        },
+        "db": {
+          "type": ["null", "string"]
+        },
+        "ssl": {
+          "type": ["null", "boolean"]
+        },
+        "let-user-control-scheduling": {
+          "type": ["null", "boolean"]
+        },
+        "use-jvm-timezone": {
+          "type": ["null", "boolean"]
+        },
+        "include-user-id-and-hash": {
+          "type": ["null", "boolean"]
+        }
+      }
+    },
+    "is_sample": {
+      "type": ["null", "boolean"]
+    },
+    "id": {
+      "type": ["null", "integer"]
```

```suggestion
      "type": "integer"
```
This field is declared as the PK